client/components/TestQueue/index.jsx

```diff
@@ -1,329 +1,255 @@
+                        show={isShowingAddToQueueModal}
+                        handleClose={handleCloseAddTestPlanToQueueModal}
+                        handleAddToTestQueue={refetch}
+                    />
                 )}
-                <DeleteResultsModal
-                    show={this.state.showDeleteResultsModal}
-                    handleClose={this.closeDeleteResultsModal}
-                    user={this.state.resultsDelete.username}
-                    run={this.state.resultsDelete.run}
-                    admin={admin}
-                    deleteResults={this.deleteResults}
-                />
             </Container>
         );
     }
-}
+
+    return (
+        <Container as="main">
+            <Helmet>
+                <title>{`Test Queue | ARIA-AT`}</title>
+            </Helmet>
+            <h1>Test Queue</h1>
+            <p data-testid="test-queue-instructions">
+                Assign yourself a test plan or start executing one that is
+                already assigned to you.
+            </p>
+
+            {isAdmin && (
+                <AddTestPlanToQueueContainer
+                    handleOpenDialog={() => enableAddToQueueModal(true)}
+                />
+            )}
+
+            {Object.keys(structuredTestPlanTargets).map(key =>
+                renderAtBrowserList(key, structuredTestPlanTargets[key])
+            )}
+
+            <DeleteResultsModal
+                show={isShowingDeleteResultsModal}
+                isAdmin={isAdmin}
+                title={deleteResultsDetails.title}
+                username={deleteResultsDetails.username}
+                handleClose={handleCloseDeleteResultsModal}
+                handleDeleteResults={handleDeleteResults}
+            />
+
+            {isAdmin && isShowingAddToQueueModal && (
+                <AddTestPlanToQueueModal
+                    show={isShowingAddToQueueModal}
+                    handleClose={handleCloseAddTestPlanToQueueModal}
+                    handleAddToTestQueue={refetch}
+                />
+            )}
+        </Container>
+    );
+};
 
 TestQueue.propTypes = {
-    ats: PropTypes.array,
-    activeRunsById: PropTypes.object,
-    activeRunConfiguration: PropTypes.object,
-    admin: PropTypes.bool,
-    usersById: PropTypes.object,
-    userId: PropTypes.number,
-    testsFetched: PropTypes.bool,
-    dispatch: PropTypes.func
+    auth: PropTypes.object
 };
 
 const mapStateToProps = state => {
-    const { activeRunsById, activeRunConfiguration } = state.runs;
-    const { usersById } = state.users;
-    const userId = state.user.id;
-    const roles = state.user.roles;
-    let testsFetched = true;
-
-    let admin = (roles || []).includes('admin');
-
-    return {
-        activeRunsById,
-        activeRunConfiguration,
-        testsFetched,
-        usersById,
-        userId,
-        admin,
-        ats: state.ats
-    };
+    const { auth } = state;
+    return { auth };
 };
```

Comment:
I was imagining that auth was now fully in the GraphQL world, but it seems there's something still stuck in Redux. Is there a way to remove this? I think I mentioned somewhere else that I was worried about how confusing it might be to have Redux and GraphQL in the same component, since removing Redux will probably happen once GraphQL is enabled everywhere.

Reply:
Think my previous comment on this covers this:

> Managing the auth session through redux is commonplace for me but if this is something easily done with Apollo here, we could remove this altogether and I'll follow your lead there.

Let me know if it doesn't